esti/s3_gateway_test.go

```diff
@@ -11,35 +11,29 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	awscredentials "github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/go-openapi/swag"
 	"github.com/minio/minio-go/v7"
```

Comment:
Why do we have both AWS and MinIO clients?

MinIO has to be here, it's the only way that I could find to for SigV2 in the test.  But if you can get rid of it -- sure, let's do that across the board.

Reply:
Still not sure why we're using two client type...

Reply:
Switched to use minio client to delete the underlying object for this test.
1. Don't think we need minio client - [issue](https://github.com/treeverse/lakeFS/issues/6838) 
2. In this test I cloud try to create an entry to non-exists physical address or delete the underlying object. The first one was preferred, but it is harder to do with our newer API.